### DISCUSSION TITLE
Bug: Command Handler Not Functioning Properly

### DISCUSSION THREAD

**Discussion by Pro0699** [eritislami/evobot#393] [Q&A]
**Describe the bug**
I Am trying to add a command filter.js but in the help its showing undifined

**How To Reproduce**
1. Add The Command filter.js 
2. Type the help command in the server [###help]
 
**Expected behavior**
Command should take place properly

**Environment (add if possible)**
* Node.js version: 12.16.1 

**Additional information & screenshots**
Add any other context or screenshots about the problem here
![image](https://user-images.githubusercontent.com/75003322/100535183-a30e9300-323c-11eb-8848-485167dc6c2e.png)
![image](https://user-images.githubusercontent.com/75003322/100535190-b15caf00-323c-11eb-9202-409a09e1ea3d.png)

**Comment by Takiyo0** [comment]
Do you use it properly? It must be like
```
module.exports = {
name: 'filter',
aliases: ["aliases here"],
execute(message) {
      //your code here

  }
}
```
or with async and args
```
module.exports = {
name: 'filter',
aliases: ["aliases here"],
async execute(message, args) {
      //your code here

  }
}
```

**Comment by eritislami** [comment]
Clearly not a bug, don't open issues for programming support.
